nixos/tests/nextcloud/basic.nix

```diff
@@ -59,6 +59,7 @@ runTest (
 
           services.nextcloud = {
             enable = true;
+            config.dbtype = "sqlite";
```

Comment:
Shouldn't this ideally also test the default?

Reply:
Hmm, fair.
I'm inclined to go for
* `basic` testing all defaults (i.e. postgresql)
* `with-sqlite-and-redis` to have Redis and sqlite coverage (it's good to have coverage for each DB and caching approach, but I don't recall a single issue in several years of maintaining this that we would've caught by testing all combinations of that).

Reply:
That sounds good to me.

Reply:
Hmm this won't work out as planned given that notify push is also part of that test and this doesn't seem to work with sqlite (though it doesn't complain when sqlite is configured and just fails at runtime cc @SuperSandro2000)

Reply:
Fixed.